noxfile.py

```diff
@@ -52,9 +52,27 @@ def set_iree_state():
     return state
 
 
+def get_build_dependencies():
+    """
+    Gets build-time dependencies from `pyproject.toml` specific to platform
+    """
+    build_dependencies = nox.project.load_toml("pyproject.toml")["build-system"][
+        "requires"
+    ]
+    if sys.platform == "win32":
+        build_dependencies = [
+            dep
+            for dep in build_dependencies
+            if not (dep.startswith("casadi") or dep.startswith("cmake"))
+        ]
```

Comment:
Question: Do we need to process the list of dependencies beforehand? The installation using `session.install("<...>")` should work equally well with [environment markers](https://packaging.python.org/en/latest/specifications/dependency-specifiers/#environment-markers).

Reply:
Yes, I tested it out, and the env markers work well without the pre-processing. I am removing it.